File: tests/fixtures/lsp-smoke/probe.ts
// LSP smoke fixture for the TypeScript / JavaScript language server.
//
// Deterministic symbols so `lsp_definition` / `lsp_hover` have stable targets,
// plus one real type error so the server publishes a diagnostic.

export function answer(): number {
  return 42;
}

const value = answer();

// Intentional undefined symbol -> real diagnostic from typescript-language-server.
console.log(undefinedSymbol);

File: tests/fixtures/lsp-smoke/probe.py
# LSP smoke fixture for the Pyright language server.
#
# Deterministic symbol `add` for definition/hover, plus one real undefined-name
# reference so Pyright publishes a diagnostic.


def add(a: int, b: int) -> int:
    return a + b


total = add(1, 2)

# Intentional undefined name -> real diagnostic from pyright.
print(undefined_name)
